src/Model/TroopManager.php

```diff
@@ -32,12 +32,15 @@ public function __construct()
 
     public function insert(Troop $troop)
     {
+
         // prepared request
-        $insert = $this->pdo->prepare("INSERT INTO " . self::TABLE . " (name, strength) VALUES (:name, :strength)");
+        $insert = $this->pdo->prepare("INSERT INTO " . self::TABLE . " (name, strength, tiredness)
```

Comment:
there's no tiredness field in the database schema provided by the defendthecastle.sql file

Reply:
Modification ok